txmongo/collection.py

```diff
@@ -394,7 +392,6 @@ def update(self, spec, document, upsert=False, multi=False, safe=None, flags=0,
             ret = yield proto.get_last_error(str(self._database), **write_concern.document)
             defer.returnValue(ret)
 
-
```

Comment:
The main think I hate about PEP8 is that it prohibits double empty lines between methods :(